Azure_Pipelines/azure-pipelines.terminology.cd.yaml

```
@@ -0,0 +1,100 @@
+  - task: DotNetCoreCLI@2
+    displayName: Build
+    inputs:
+      command: build
+      projects: '$(project)'
+
+  - task: DotNetCoreCLI@2
+    inputs:
+      command: 'test'
+      projects: $(testProject)'
+    displayName: 'Run Tests'
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)